### PR TITLE
docs: update readme with contribution instructions

### PR DESCRIPTION
**Pull request by fhennig**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -55,55 +71,44 @@ Target repositories are configured in `repositories.yaml` in the following form:
...
-Anything that is listed here will be deleted from the target repositories.
+If you want to make a change that should be rolled out to all operators, make the change in the `template` directory.
+Consult the section above to learn more about the structure of the template.
 
-NOTE: Deletion is the last step that is performed, so if there is an overlap between files existing in the template folder and this setting, the files would not be rolled out, since they'd get deleted before creting the pull request.
+=== Test changes locally
 
-=== Local
-While this can in theory also be used locally this has not yet been tested and may or may not be documented and supported here in the future.
+1. Create the directory `work`, clone all (or some) operators into this directory, and run the `test.sh` script.
```
Comment (NickLarsenNZ) [inline README.adoc:88]: I believe it will clone the repos for you. So you only have to adjust the list in the repositories.yaml.

**Review by NickLarsenNZ — APPROVED** [review]
LGTM

File: README.adoc (hunk 1 above)
Comment (fhennig) [reply README.adoc:88]: Oh yeah right. Lemme check and then I'll update this (it was copy pasted from other seemingly slightly outdated instructions)

Comment (fhennig) [reply README.adoc]: @NickLarsenNZ I've changed it now!

**Review by NickLarsenNZ — APPROVED** [review]
LGTM

**Pull request merged**